kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -365,9 +365,10 @@ object Fiber extends FiberPlatformSpecific:
             def onComplete(f: Result[E, A] => Unit)(using AllowUnsafe): Unit                             = self.onComplete(f)
             def onInterrupt(f: Panic => Unit)(using Frame): Unit                                         = self.onInterrupt(f)
             def block(deadline: Clock.Deadline.Unsafe)(using AllowUnsafe, Frame): Result[E | Timeout, A] = self.block(deadline)
-            def interrupt(error: Panic)(using AllowUnsafe): Boolean                                      = self.interrupt(error)
-            def interruptDiscard(error: Panic)(using AllowUnsafe): Unit                                  = discard(self.interrupt(error))
-            def mask()(using AllowUnsafe): Unsafe[E, A]                                                  = self.mask()
+            def interrupt()(using frame: Frame, allow: AllowUnsafe): Boolean = self.interrupt(Panic(Interrupted(frame)))
```

Comment:
A new overloaded `interrupt` without params. The following methods were changed only because of formatting

Reply:
This will be useful! I think kyo-zio can use it.